spec/factories/api_opportunity.rb

```diff
@@ -2,10 +2,22 @@
 
 FactoryBot.define do
   factory :api_opportunity, class: Opportunity do
-    salesforce_id {Faker::Alphanumeric.alphanumeric(number: 10, min_alpha: 3, min_numeric: 3)}
+    salesforce_id { '0067X000007eQMeQAM' }
```

Comment:
how does this affect creating multiple records with factorybot?

Reply:
@mwharrison Since the id does not need to be unique, it works. VCR needs real ids to record the tests.